packages/liveblocks-node/src/client.ts

```diff
@@ -929,6 +931,205 @@ export class Liveblocks {
     }
     return (await res.json()) as Promise<CommentData>;
   }
+
+  /**
+   * Creates a comment.
+   *
+   * @param params.roomId The room ID to create the comment in.
+   * @param params.threadId The thread ID to create the comment in.
+   * @returns The created comment.
+   */
+  public async createComment(params: {
+    roomId: string;
+    threadId: string;
+    comment: {
+      userId: string;
+      /**
+       * ISO 8601 date string
+       */
+      createdAt?: string;
+      body: CommentBody;
+    };
+  }): Promise<CommentData> {
+    const { roomId, threadId, comment } = params;
+
+    const res = await this.post(
+      url`/v2/rooms/${roomId}/threads/${threadId}/comments`,
+      comment
+    );
+    if (!res.ok) {
+      const text = await res.text();
+      throw new LiveblocksError(res.status, text);
+    }
+    return (await res.json()) as Promise<CommentData>;
```

Comment:
`CommentData` includes a `createdAt`, and may include `editedAt` and `deletedAt` field. The `createComment` currently doesn't convert them into a `Date` object before returning them. We should consider doing that, as we already do that for other methods like `getRoom`, `createRoom`, etc.

Reply:
@FlowFlorent I've created two helper methods: `convertToCommentData` and `convertToThreadData` that takes the response returned by the API and massages this object so that the date strings are formatted as Date objects, which are easier to consume by clients. These methods are currently used by `editComment`, `createThread`, `editThread`. We might want to do the same for `createComment`.

https://github.com/liveblocks/liveblocks/blob/143e3288f7c47e1929adb73cf6165a98258cfe94/packages/liveblocks-node/src/client.ts#L217-L236

https://github.com/liveblocks/liveblocks/blob/143e3288f7c47e1929adb73cf6165a98258cfe94/packages/liveblocks-node/src/client.ts#L245-L259

Reply:
I ended up updating the the `createComment` method to use the `convertToCommentData`. See https://github.com/liveblocks/liveblocks/pull/1325#discussion_r1413584082

Reply:
This is really nice! Good call on the abstraction.